docs/ldml/tr35.md

```diff
@@ -4365,8 +4370,31 @@ Other contributors to CLDR are listed on the [CLDR Project Page](https://www.uni
 ### [Inheritance](https://unicode.org/reports/tr35/dev/tr35.html#inheritance-marker)
 * Documented the use of Inheritance Marker (↑↑↑) in the CLDR repository
 
-### [MessageFormat](tr35-messageFormat.html#Contents)
-_TBD: Note: we should also add below to the differences in 46.1; can get from release notes for 46.1_
+### [MessageFormat](tr35-messageFormat.md#Contents)
+- Made MessageFormat stable and made the stability policy normative.
+- Added or clarified terminology:
+  - Linked all terminology in the MessageFormat section to the term's definition.
+  - Defined _option value_, _literal key_, _string value_, and _digit size option_ formally. Other definitions were defined or modified for clarity.
+  - Replaced the concept of a "function registry" with default functions and `u:` namespace functions and options.
+  - Changed the data model to be called the "Interchange Data Model"
+- Modified portions of the syntax (ABNF)
+  - Removed `number-literal` from the ABNF and moved its definition to the _default functions_ for numeric formatting.
+  - Changed (expanded) the range of characters allowed in an unquoted literal and simplified syntax character set definitions.
+- Revised the _Default Bidi Strategy_.
+- Enabled functions to know whether an _option value_ was set using a literal or a variable, which is necessary for some function's selection mechanism (see below).
+- Updated the _default functions_:
+  - Only three _default functions_ are stable: `:string`, `:number`, and `:integer`. Other functions are Draft.
+  - Some options have been removed, modified, or made optional.
+  - The `select` option on `:number` and `:integer` functions now accepts only literal values.
+  - Removed the `style` `percent` from `:number` and `:integer` pending future standardization.
+  - The `u:` namespace options `u:id` and `u:dir` are optional; `u:locale` is Draft.
+- Clarified how to include the character "*" in a key.
+
+In addition, the following changes were made in v46.1:
+- Removed all of the reserved and private use syntax constructs, simplifying the grammar.
+- Changed the structure of the `.match` (selector) to require use of `.local` or `.input` declarations. This is a breaking change for existing messages.
```

Comment:
```suggestion
- Changed the structure of the `.match` (selector) to require use of `.local` or `.input` declarations.
```
We shouldn't say the 'breaking change' because we don't have any conformant implementations (because nobody could be).

If we did want to say something, it would be a general note that would apply to the 47 changes also. But I think we can just omit.